test/chainsaw/testanalysis/analysis-controller-existing-status/00-install.yaml

```
@@ -0,0 +1,70 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: testy
+---
+apiVersion: metrics.keptn.sh/v1beta1
+kind: AnalysisValueTemplate
+metadata:
+  name: ready
+  namespace: testy
+spec:
+  provider:
+    name: my-mocked-provider
+    namespace: testy
+  query: 'sum(kube_pod_container_status_ready{namespace="{{.ns}}"})'
+---
+apiVersion: metrics.keptn.sh/v1beta1
+kind: AnalysisDefinition
+metadata:
+  name: ed-my-proj-dev-svc1
+  namespace: testy
+spec:
+  objectives:
+    - analysisValueTemplateRef:
+        name: ready
+        namespace: testy
+      target:
+        failure:
+          lessThan:
+            fixedValue: 2
+        warning:
+          lessThan:
+            fixedValue: 3
+      weight: 1
+      keyObjective: false
+  totalScore:
+    passPercentage: 90
+    warningPercentage: 75
+---
+apiVersion: metrics.keptn.sh/v1beta1
+kind: Analysis
+metadata:
+  name: analysis-sample
+  namespace: testy
+spec:
+  timeframe:
+    from: 2023-09-14T07:33:19Z
+    to: 2023-09-14T08:33:19Z
+  args:
+    "ns": "keptn-system"
+  analysisDefinition:
+    name: ed-my-proj-dev-svc1
+    namespace: testy
+status:
+  storedValues:
+    my-provider-query-1:
+      objectiveReference:
+        name: objective-template-1
+      value: 1
+      errMsg: ""
+
+---
+apiVersion: metrics.keptn.sh/v1beta1
+kind: KeptnMetricsProvider
+metadata:
+  name: my-mocked-provider
+  namespace: testy
+spec:
+  type: prometheus
+  targetServer: "http://mockserver.testy.svc.cluster.local:1080"
```

Here it might be possible to use the $NAMESPACE variable (not sure though, please correct me if I am wrong)

```suggestion
  targetServer: "http://mockserver.$NAMESPACE.svc.cluster.local:1080"
```

---

but it should be possible, see here https://github.com/keptn/lifecycle-toolkit/pull/2961/files#diff-d4c6622ff7eaa631929eef8c34792be7c3728343373f1161ff3cd9184e223382R104

---

It's only possible if you call `envsubst`